### PR TITLE
fix(banners): remove unnecessary banners INNO-841

### PR DESCRIPTION
**Pull request by raduchiriac**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [x] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
- [x] I have checked the dependencies
- [x] I have given the fractal status “ready” to my component
- [x] I have declared `@define mycomponent` in the SCSS file
- [x] I have specified `margin: 0;` on the CSS component
- [x] I have provided tests
- [x] I follow the naming guidelines
- [x] the component supports composition
- [x] there are no hardcoded strings (all content come from the context)
- [x] I have filled the README.md file (at least a few lines)
- [x] My component is listed in the root README
- [x] My PR has the right label(s)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview ready!

Built with commit 5998399f02e1b2a45252320a9973cf29677cdbc4

https://deploy-preview-493--europa-component-library.netlify.com

File: framework/components/ecl-banners/README.md [hunk 1]
```diff
@@ -27,30 +21,18 @@ The banner is used for optional, mainly promotional material.
...
-- do not use more than one background banner or video banner on a page
 - do not use more than two quote banners on a page
 
 ## Technical details
 
 The banner itself is not a standalone component, one of the following modifiers should always be used:
 
-- `ecl-banner--background`: banner with background-image.
-- `ecl-banner--video`: banner with video and description. Based on ecl-files component.
 - `ecl-banner--quote`: banner with a quote. Based on ecl-blockquotes component.
```
Comment (emeryro) [inline framework/components/ecl-banners/README.md]: As there is only one modifier left, I guess we can apply the css style directly to it (i.e. remove the "ecl-banner--quote").
And so the last paragraph in the documentation may be removed too.

File: framework/components/ecl-banners/ecl-banners.config.js [hunk 2]
```diff
@@ -12,23 +12,9 @@ module.exports = {
       <!-- End: @${item.handle} -->\n
     `;
   },
-  default: 'background',
+  default: 'quote',
   variants: [
```
Comment (emeryro) [inline framework/components/ecl-banners/ecl-banners.config.js]: We can even remove the variants, as there is only possibility left.
So config file could looks like this:
module.exports = {
  title: 'Banners',
  label: 'Banners',
  status: 'ready',
  tags: ['molecule'],
  context: {
    quote: {
      body: '
        Nullam eget gravida sapien. Curabitur nec ligula varius tellus blandit auctor in non felis.
        Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae;
        Pellentesque in volutpat eros, ut fringilla tortor.
      ',
      author: 'President Juncker',
    },
  },
};

File: framework/components/ecl-banners/ecl-banners.twig [hunk 3]
```diff
@@ -1,22 +1,14 @@
...
-    - "src" (string)
-    - "title" (object):
-      - "href" (string)
-      - "label" (string)
-    - "description" (string): main content (if type == "background" or "video")
-    - "background_description_block" (block): main content (if type == "background")
-    - "video" (object): parameters passed to ecl-files (if type == "video")
-    - "video_description_block" (block): main content (if type == "video")
-    - "quote" (object): parameters passed to ecl-blockquotes (if type == "quote")
+    - "type" (string): "quote"
```
Comment (emeryro) [inline framework/components/ecl-banners/ecl-banners.twig]: We could remove this parameter if it can take only one value

**Pull request merged**